chpl/chpl-service/src/main/java/gov/healthit/chpl/entity/CriterionProductStatisticsEntity.java

```diff
@@ -0,0 +1,130 @@
+    package gov.healthit.chpl.entity;
+
+import java.io.Serializable;
+import java.util.Date;
+
+import javax.persistence.Basic;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.Transient;
+
+/**
+ * Entity object representing the criterion_product_statistics table.
+ * @author alarned
+ *
+ */
+@Entity
+@Table(name = "criterion_product_statistics")
+public class CriterionProductStatisticsEntity implements Serializable {
+    private static final long serialVersionUID = -4258273713908999510L;
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Basic(optional = false)
+    @Column(name = "id", nullable = false)
+    private Long id;
+
+    @Basic(optional = false)
+    @Column(name = "product_count", nullable = false)
+    private Long productCount;
+
+    @Basic(optional = false)
+    @Column(name = "certification_criterion_id", nullable = false)
+    private Long certificationCriterionId;
```

Comment:
Do you also want to join in the Certification Criterion here so it's easy to pull out the "170.315"-ish name? That must be done elsewhere and I just missed it.

Reply:
I am retrieving the name somewhere, but I can't recall where. I had the impression entities were supposed to match the database table structure, which doesn't have the name in it. Am I mistaken with that?